### PR TITLE
[ELY-2473] Move method getEntry from KeyStoreBackedSecurityRealm to its inner class KeyStoreRealmIdentity

### PR DESCRIPTION
**Pull request by albertotn**
See https://issues.redhat.com/browse/ELY-2473

### CODE REVIEW

File: auth/realm/base/src/main/java/org/wildfly/security/auth/realm/KeyStoreBackedSecurityRealm.java [hunk 1]
```diff
@@ -211,5 +198,18 @@ public boolean verifyEvidence(final Evidence evidence) throws RealmUnavailableEx
...
+              KeyStore.Entry entry = keyStore.getEntry(name, null);
+              if (entry == null) {
+                  log.tracef("KeyStoreRealm: alias [%s] does not exist in KeyStore", name);
+              }
+              return entry;
+          } catch (NoSuchAlgorithmException | UnrecoverableEntryException | KeyStoreException e) {
+              log.tracef(e, "KeyStoreRealm: Obtaining entry [%s] from KeyStore failed", name);
+              return null;
+          }
+      }
```
Comment (skyllarr) [inline auth/realm/base/src/main/java/org/wildfly/security/auth/realm/KeyStoreBackedSecurityRealm.java]: @albertotn Thank you for the PR! Just a minor, this line and the line 201 has trailing spaces that needs to be removed in order for the style check to pass.

Comment (albertotn) [reply auth/realm/base/src/main/java/org/wildfly/security/auth/realm/KeyStoreBackedSecurityRealm.java]: I've updated, now running checkstyle:checkstyle it's fine for me

**Comment by skyllarr** [comment]
@albertotn Thank you, please squash the 2 commits into 1 and update the commit message to be: `[ELY-2473] Move method getEntry from KeyStoreBackedSecurityRealm to its inner class KeyStoreRealmIdentity`. If you are not sure how to do that in git just let us know

**Comment by albertotn** [comment]
now it should be fine ?

**Comment by skyllarr** [comment]
@albertotn Please squash the 4 commits into 1 needed commit. You can do that by using interactive rebase, e.g: git rebase -i HEAD~4

**Comment by albertotn** [comment]
Closed see PR https://github.com/wildfly-security/wildfly-elytron/pull/1825

**Pull request closed without merging**